core/templates/dev/head/forms/formBuilder.js

```diff
@@ -888,6 +888,14 @@ oppia.config(['$provide', function($provide) {
       return taOptions;
     }
   ]);
+
+  $provide.decorator('$sanitize', function($delegate) {
```

Comment:
I think it is safer to do this work in the `sanitizeHtmlForRte` filter, instead of decorating `$sanitize`. Decorating `$sanitize` affect every usage of `$sanitize` anywhere in the application, but we only want this to functionality for the RTE.